### DISCUSSION TITLE
Copilot plugin for pet text editor

### DISCUSSION THREAD

**Discussion by Davidy22** [community/community#10664] [Copilot Conversations]
I recently gained access to copilot and I see in the readme there are only getting started guides for three editors that I don't use. I want to implement a plugin for the editor that I use daily, is there anything I should know before I go source diving into the existing implementations for inspiration?

**Comment by phyllisstein** [comment]
Yes, I have a similar question about Panic’s [Nova](https://nova.app) and my problematic fave Sublime Text. Do you expect to publicize the API at some point? I’ve implemented a number of Sublime packages, and wouldn’t mind exercising those muscles again. Nova is new to me, but I’ve been looking for an excuse to learn their internals.
